barretenberg/cpp/src/barretenberg/sumcheck/sumcheck.hpp

```diff
@@ -7,6 +7,111 @@
 
 namespace bb {
 
+/*! \brief The implementation of the sum-check Prover for the statements of the form \f$\sum_{\vec \ell \in \{0,1\}^d}
+pow_{\beta}(\vec \ell) \cdot F \left(P_1(\vec \ell),\ldots, P_N(\vec \ell) \right)  = 0 \f$ for multi-linear polynomials
+\f$P_1, \ldots, P_N \f$.
+
+   \details
+ ## Notation and Setup
+
+ ### Obtaining Prover/Honk Polynomials
+ The Sumcheck is applied to a \p Flavor that in its turn specifies the information about the multi-variate polynomials
```

Comment:
```suggestion
The sumcheck protocol  is applied to a \p Flavor that in turn specifies the information about the multi-variate polynomials
```

Reply:
So I think this is a bit wrong. Sumcheck is applied to some multivariate polynomials and the exact polynomials to which it is applied are specified in the Flavor. Will you rephrase this a bit please?